Analysis/Tasks/jetfinder.cxx

```diff
@@ -32,42 +34,85 @@ struct JetFinderTask {
   Produces<o2::aod::Jets> jets;
   Produces<o2::aod::JetConstituents> constituents;
 
+  // options for jet finding
+  Configurable<float> rParam{"rParam", 0.4, "jet radius"};
+  Configurable<float> ghostEtamax{"ghostEtamax", 1.0, "eta max for ghosts"};
+  Configurable<float> minJetPt{"minJetPt", 0., "minimum jet pt"};
+  // TODO: check if the initialization is sound
+  Configurable<float> maxJetEta{"maxJetEta", 0.9f - rParam, "eta range for jets"};
```

Comment:
@ktf Initialising a configurable based on the value of another configurable does not seem to work (or rather it uses the default value of `rParam` here, even if overridden on the command line). Can this possibly work? If not, how can I distinguish if a configurable is left at the default or set to the same value as the default explicitly on the command line?

Reply:
For this to work we would need a `Configurable<Expression>` which is not there at the moment. Without that the 0.9f - rParam is calculated at start time (before the command line is interpreted).

Reply:
@aalkin is this what you had in mind during the meeting?

Reply:
@ktf yes, and later I found that it is trickier than I expected - I will need to make configurables use binding nodes behind the scenes so that we can have "lazy evaluated" expressions like that.

Reply:
@ktf @aalkin Thanks! Then, I'll change that to a fixed initialisation for now and leave a TODO comment that this is to be changed once lazy evaluation is supported.